### PR TITLE
Case screen fix for call tasks

### PR DESCRIPTION
**Pull request by GPaoloni**
Fixed a bug in timeline component where call tasks were causing infinite rerenders.

Pending: render the proper color icon for calls. For this we might have to send a supoport ticket, as default call channel properties differs from the others.

### CODE REVIEW

File: plugin-hrm-form/src/components/case/TimelineIcon.jsx [hunk 1]
```diff
@@ -5,39 +5,50 @@ import * as Flex from '@twilio/flex-ui';
...
-
-  return <TimelineIconContainer>{icon}</TimelineIconContainer>;
 };
 
+const TimelineIcon = ({ type }) => <TimelineIconContainer>{getIcon(type)}</TimelineIconContainer>;
+
 TimelineIcon.displayName = 'TimelineIcon';
 TimelineIcon.propTypes = {
-  type: PropTypes.oneOf(['whatsapp', 'facebook', 'web', 'sms', 'note']).isRequired,
+  type: PropTypes.oneOf(Object.values(channelTypes)).isRequired,
```
Comment (nickhurlburt) [inline plugin-hrm-form/src/components/case/TimelineIcon.jsx:33]: Is this right?  It's missing `'note'`.

Comment (GPaoloni) [reply plugin-hrm-form/src/components/case/TimelineIcon.jsx:33]: Oh, good catch, thanks.

**Comment by murilovmachado** [comment]
deployed to staging.

**Comment by nickhurlburt** [comment]
Yay!

**Pull request merged**